rust-runtime/aws-smithy-http-server/Cargo.toml

```diff
@@ -20,11 +20,22 @@ aws-smithy-http = { path = "../aws-smithy-http" }
 aws-smithy-types = { path = "../aws-smithy-types" }
 aws-smithy-json = { path = "../aws-smithy-json" }
 axum = { version = "0.3", features = [ "http1", "http2", "headers", "mime", "tower-log" ], default-features = false }
+async-trait = "0.1"
 bytes = "1.1"
+futures-util = { version = "0.3", default-features = false }
 http = "0.2"
 http-body = "0.4"
 hyper = { version = "0.14", features = ["server", "http1", "http2", "tcp"] }
 mime = "0.3"
+pin-project-lite = "0.2.7"
+regex = "1.0"
+serde_urlencoded = "0.7"
+thiserror = "1"
+tokio = { version = "1.0", features = ["full"] }
```

Comment:
Tokio is now at 1.13.0. Any reason to not use "1" so we get the latest stable version?

Reply:
Isn't the behavior the same? From https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements

> An update is allowed if the new version number does not modify the left-most non-zero digit in the major, minor, patch grouping.

Reply:
There is no caret here, it's just "1.0" so for what I understand no, the behaviour is not the same.

Reply:
From https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio

> The string "0.1.12" is a semver version requirement. Since this string does not have any operators in it, it is interpreted the same way as if we had specified "^0.1.12", which is called a caret requirement.

Reply:
Ah! Ok, forget what I said here.